clmm/dataops/__init__.py

```diff
@@ -376,12 +389,16 @@ def _compute_lensing_angles_flatsky(ra_lens, dec_lens, ra_source_list, dec_sourc
         phi[angsep == 0.0] = 0.0
     else:
         phi = 0.0 if angsep == 0.0 else phi
+    if coordinate_system == "sky":
+        phi = np.pi - phi
```

Comment:
`phi = np.arctan2(deltay, deltax)` should do the work. (note: for coordinate_system == "pixel", the calculation is `phi = np.arctan2(deltay, -deltax)`)